### ISSUE TITLE
Discussion: scikit-bio sprint at EuroSciPy?

### ISSUE THREAD

**Issue by jvkersch** [biocore/scikit-bio#1573]
Apologies if this is not the right channel, but I'm wondering if there's any interest in having a scikit-bio sprint at EuroScipy later this year (August 28-September 1, Trento, Italy). EuroSciPy is (as the name suggests) the European version of the SciPy conference, and previous installments have seen sprints on Numpy, Scipy, Pandas, scikit-learn, scikit-image, and more, so chances are there would be a good crowd of interested people. Perhaps any of the existing scikit-bio contributors are even planning to attend this conference? 

Any thoughts? Please feel free to migrate this issue to  a mailing list, or to another discussion forum.

**Comment by ElDeveloper** [comment]
I don't know of any of the core contributors who will be attending EuroSciPy, but I think it would be a good idea if something like this happened.

Just in case, I've twitted about this: https://twitter.com/yoshikivb/status/1017461290665574401

**Comment by jvkersch** [comment]
Thanks @ElDeveloper !

Just in case if anybody is at _SciPy_ in Austin right now, I would love to catch up and make this more concrete!

**Comment by ElDeveloper** [comment]
I am not aware of anyone from the core group there either. Have fun in 
Austin!

**Comment by jvkersch** [comment]
This can be closed now that EuroSciPy is over. For the record, I gave a tutorial about scikit-bio (among other things) which was relatively well received and attracted a lot of questions about the package. My tutorial files are at https://github.com/jvkersch/bio-tutorial but they reflect how a computational person thinks about bio-informatics and are probably not fit for public consumption. I also got to work on https://github.com/biocore/scikit-bio/issues/254.
